src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -16,28 +18,41 @@ export default function ColorPickerControl({
     ...rest
 }: Props): JSX.Element | null {
     const [isColorPickerToggled, setIsColorPickerToggled] = useState(false);
+    const [isPaletteActive, handlers] = useAttention();
 
     const handleSelect = (color: string): void => {
         setIsColorPickerToggled(false);
         onColorSelect(color);
     };
 
+    const handleBlur = (): void => {
+        if (isPaletteActive) {
+            return;
+        }
+        setIsColorPickerToggled(false);
+    };
+
+    const handleClick = (): void => setIsColorPickerToggled(!isColorPickerToggled);
+
     return (
         <div className="bp-ColorPickerControl">
             <button
                 className="bp-ColorPickerControl-button"
                 data-testid="bp-ColorPickerControl-button"
-                onClick={(): void => setIsColorPickerToggled(!isColorPickerToggled)}
+                onBlur={handleBlur}
+                onClick={handleClick}
                 type="button"
                 {...rest}
             >
                 <div className="bp-ColorPickerControl-swatch" style={{ backgroundColor: activeColor }} />
             </button>
-            {isColorPickerToggled && (
-                <div className="bp-ColorPickerControl-palette">
-                    <ColorPickerPalette colors={colors} data-testid="bp-ColorPickerPalette" onSelect={handleSelect} />
-                </div>
-            )}
+            <div
```

Comment:
This changes the tab behavior so that the colors are included in the tab order even when the picker is closed. Is that desirable? It seems like we'd want to keep the `isColorPickerToggled &&` condition.

Reply:
Actually, I'm mistaken, since `display: none` will cause them to be ignored, unlike other visual-only hiding techniques. This should work fine.